packages/theme/src/colors.ts

```diff
@@ -0,0 +1,65 @@
+import type * as CSS from 'csstype';
+
+const defaultColorVariant = {
+    green: '#00854D',
+    green20: 'rgba(0, 133, 77, 0.2)',
```

Comment:
What do those numbers mean?

Reply:
Not sure which part you mean, if you mean name `green20` it same name as it is used in Figma, which I think we should keep same. If you mean `rgba` it's same like just `rgb` but with alpha channel.

Reply:
I think `20` in name means 20% opacity

Reply:
Oh, pardon, I meant numbers in the name ofc haha 😅 Yeah, I thought it's from figma but I am not sure if those names are easy to remember and recognise. But that's not a comment towards your work, just a general observation.

Reply:
I think names are good in general because it coresponds with patter used for `grey`. I can imagine we will have to remember fifty shades of grey 😄

Reply:
My favorite part of coding 🤝